website/project/views/wiki.py

```diff
@@ -0,0 +1 @@
+x`
```

Comment:
Resubmitted. I ran tests after the changes locally -- weird. Sorry about
that.

On Sat, Feb 14, 2015 at 3:14 PM, Steven Loria <notifications@github.com>
wrote:

> In website/project/views/wiki.py
> <https://github.com/CenterForOpenScience/osf.io/pull/1801#discussion_r24716989>
> :
>
> > @@ -0,0 +1 @@
> > +x`
>
> This is causing the build to fail. Can you remove this?
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/CenterForOpenScience/osf.io/pull/1801/files#r24716989>
> .
>



-- 
*Harry Rybacki* - Developer, Center for Open Science
Research: http://scinet.osf.io
Blog: http://rybacki.io
[image: http://github.com/harryrybacki]
<http://github.com/HarryRybacki> [image:
https://twitter.com/HarryRybacki] <https://twitter.com/HarryRybacki> [image:
http://www.linkedin.com/in/HarryRybacki/]
<http://www.linkedin.com/in/HarryRybacki/> [image:
https://plus.google.com/+HarryRybacki]
<https://plus.google.com/+HarryRybacki>